### PR TITLE
replace GH username to PANW usename for oncall

### PR DESCRIPTION
**Pull request by israelpoli**

### CODE REVIEW

**Review by ShacharKidor — APPROVED** [review]
cool.
@kobymeir - I think this is a good option since we cannot add comments to this json. Is there a chance that the new comment key will cause a problem?

File: .github/content_roles.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {
+    "COMMENT_FORMAT": {
+        "ALL_FIELDS": "The GitHub username",
+        "ON_CALL_DEVS": "The PANW account username",
+    },
```
Comment (kobymeir) [inline .github/content_roles.json:5]:
```suggestion
    "__comment__": {
        "CONTRIBUTION_REVIEWERS": "A list of GitHub username for the contribution team reviewers pool",
        "ON_CALL_DEVS": "A list of on-call users, Use PANW account username",
        "DOC_REVIEWER": "The GitHub username for documentation reviews owner",
        "TIM_REVIEWER": "The GitHub username for TIM reviews owner"
    },
```

File: .github/content_roles.json [hunk 2]
```diff
@@ -9,7 +13,7 @@
     "ON_CALL_DEVS": [
         "skidorball",
         "ayousef",
-        "israelpoli"
+        "ipolishuk"
```
Comment (kobymeir) [inline .github/content_roles.json:16]: i asked to remove ayousef.

**Review by kobymeir — CHANGES_REQUESTED** [review]

**Comment by kobymeir** [comment]
@ShacharKidor yes, that's the right approach, also used in other places in the system.

**Review by kobymeir — APPROVED** [review]

**Comment by israelpoli** [comment]
Force merge - only `content_roles.json` changed
doesn't affect anything

**Pull request merged**